packages/shared/lib/platform.ts

```diff
@@ -1,564 +1,5 @@
-import { Electron } from 'shared/lib/electron'
-import { CapacitorApi } from '../../mobile/capacitor/capacitorApi'
 import { IPlatform, Platforms } from './typings/platform'
-import * as binding from '../../mobile/capacitor/walletPluginApi'
-import type { AppSettings } from './typings/app'
 
-const PLATFORM = process.env.PLATFORM
+const IS_MOBILE = process.env.PLATFORM == Platforms.MOBILE
```

Comment:
```suggestion
const IS_MOBILE = process.env.PLATFORM === Platforms.MOBILE
```

We should probably start using `===` everywhere for consistency and safety

Reply:
Ah yes, I'll make this change. I'll also add to coding conventions and linting 👍